features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that deletes the previous and current branches
```

Comment:
Feature: setting "main" as the previous Git branch after running a Git Town command that deletes the previous and current branch

?